app/Notifications/CommentNotification.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Comment;
+use Illuminate\Bus\Queueable;
+use Illuminate\Notifications\Messages\MailMessage;
+use Illuminate\Notifications\Notification;
+
+class CommentNotification extends Notification
+{
+    use Queueable;
+
+    private Comment $comment;
+
+    /**
+     * Create a new notification instance.
+     *
+     * @return void
+     */
+    public function __construct(Comment $comment)
+    {
+        $this->comment = $comment;
+    }
+
+    /**
+     * Get the notification's delivery channels.
+     *
+     * @param mixed $notifiable
+     * @return array
+     */
+    public function via($notifiable)
+    {
+        return ['mail'];
+    }
+
+    /**
+     * Get the mail representation of the notification.
+     *
+     * @param mixed $notifiable
+     * @return \Illuminate\Notifications\Messages\MailMessage
+     */
+    public function toMail($notifiable)
+    {
+        return (new MailMessage)
+            ->line('There are new comments on a booking.')
```

Comment:
Add more context to the notification. What booking is this for? Why are they receiving this notification (Are they a reviewer? the submitter?) ? 

If a user has many bookings, this email notification is not useful or informative.

Reply:
done